### DISCUSSION TITLE
How to conditionally include the "edit this page" and "report an issue" sidebars?

### DISCUSSION THREAD

**Discussion by boshek** [quarto-dev/quarto-cli#6479] [Q&A] [answered]
### Description

This is a great feature:

![image](https://github.com/quarto-dev/quarto-cli/assets/18472598/f3c19f9e-418e-4a18-af5c-f58e98e82270)

But does anyone know how to turn it off? For example, is it possible if I had a website to include it on certain pages and not on others? Googling is failing me because searching "edit this page" brings up nearly all of the quarto docs :)

**Comment by mcanouil** [comment]
I don't believe it's possible as it currently is a project option, see <https://quarto.org/docs/websites/website-navigation.html#github-links>.

Could you elaborate on your use case? I don't see what's the benefits as if a page is "editable" on GitHub, this means all the other pages are, with or without the links.

**Reply by lostmygithubaccount** [reply]
The use case is it makes sense for every page *except* the index/home page. [We're porting over our Ibis documentation to Quarto](https://github.com/ibis-project/ibis/issues/6814) and I think the home page would look cleaner without those links at the bottom

<img width="1674" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/54814569/f76406df-d3c2-434f-934f-5cee81c0aa03">

I do want it on every other page though.

**Reply by jimjam-slam** [reply]
If you wanted to be sneaky, you could hide those links on the home page by adding the following CSS using the [`css` key](https://quarto.org/docs/reference/formats/html.html#format-options) just on the home page's frontmatter: 

```
#toc-actions {
  display: none;
}
```

**Reply by dragonstyle** [reply, marked as answer]
Starting in Quarto 1.4, you'll be able to specify `repo-actions: false` in the yaml block for a page to prevent the display of repository actions on a page (added here e8d348e938fd7be93eb41e81ed24be733edaee3b).

**Reply by lostmygithubaccount** [reply]
appreciate the quick turnaround! just confirmed it works w/ the dev version of quarto for me
